src/app/space/create/deployments/deployments.module.ts

```diff
@@ -9,6 +9,9 @@ import { TooltipModule } from 'ngx-bootstrap/tooltip';
 
 import { DeploymentsComponent } from '../deployments/deployments.component';
 import { DeploymentCardComponent } from '../deployments/components/deployment-card/deployment-card.component';
+import { DeploymentsDonutComponent } from './deployments-donut/deployments-donut.component';
+import { DeploymentsDonutChartComponent }
+  from './deployments-donut/deployments-donut-chart/deployments-donut-chart.component';
```

Comment:
This is probably just another line-length autoformatting thing but it looks pretty weird...

Reply:
The tslint has max line length of 120; I'm not sure what the standard of where to linebreak is in these scenarios. If you have a suggestion let me know.

Reply:
`import {
  DeploymentsDonutChartComponent
} from './deployments-donut/deployments-donut-chart/deployments-donut-chart.component';`

is probably short enough and fits in with style found elsewhere